utils/create-database.js

```diff
@@ -0,0 +1,82 @@
+// utils/create-database.js
```

Comment:
All the comments could be removed from this file, as theyre just describing how the various parts work rather than why, which would be way more useful in the case of complicated code